spark-examples/README.md

```diff
@@ -0,0 +1,20 @@
+# Spark Connector Examples for Pravega on SDP
+
+The battery of code examples to demonstrate the capabilities of Pravega as a data stream storage system for Apache Spark.
+
+## Deployment
+
+1. Create a project on SDP and upload the maven artifacts for [spark-connectors](https://github.com/pravega/spark-connectors) and [pravega-keycloak](https://github.com/pravega/pravega-keycloak) jar
```

Comment:
Provide two methods:
1. Download pre-built artifacts from jfrog and/or https://repo1.maven.org/maven2/io/pravega/pravega-keycloak-client/0.8.0/.
2. Git clone and gradlew install.